src/instruments/src/ND/elements/FlightPlan.tsx

```diff
@@ -52,28 +64,37 @@ export const FlightPlan: FC<FlightPathProps> = ({ x = 0, y = 0, flightPlanManage
     return null;
 };
 
-const Waypoint: FC<{ waypoint: WayPoint, mapParams: MapParameters }> = ({ waypoint, mapParams }) => {
+const Waypoint: FC<{ waypoint: WayPoint, index: number, mapParams: MapParameters, debug: boolean }> = ({ waypoint, index, mapParams, debug = false }) => {
     const [x, y] = mapParams.coordinatesToXYy(waypoint.infos.coordinates);
 
     return (
         <Layer x={x} y={y}>
-            <rect x={-4} y={-4} width={8} height={8} stroke="#00ff00" strokeWidth={2} transform="rotate(45 0 0)" />
+            <rect x={-6} y={0} width={8} height={8} stroke="#00ff00" strokeWidth={2} transform="rotate(45 4 4)" />
```

Comment:
This should be -4, -4 and rotated about 0, 0, because 0, 0 is the location of the centre of the waypoint/navaid. This will offset the waypoint symbol from it's correct location.